File: VirtualKeyboard/VirtualKeyboardVBNet/MyTableLayoutPanel.vb
Imports System.Linq
Imports System.Text

Namespace CSSoftKeyboard
	Public Class MyBufferedTableLayoutPanel
		Inherits TableLayoutPanel
		Public Sub New()
			MyBase.New()
			Me.SetStyle(ControlStyles.UserPaint Or ControlStyles.AllPaintingInWmPaint Or ControlStyles.OptimizedDoubleBuffer, True)
		End Sub
	End Class
End Namespace

File: VirtualKeyboard/VirtualKeyboardCSharp/MyTableLayoutPanel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace CSSoftKeyboard
{
    public class MyBufferedTableLayoutPanel : TableLayoutPanel
    {
        public MyBufferedTableLayoutPanel() : base()
        {
            this.SetStyle(ControlStyles.UserPaint | ControlStyles.AllPaintingInWmPaint | ControlStyles.OptimizedDoubleBuffer, true);
        }
    }
}
